File: Domain/Entities/GameResult.cs
using Domain.Common;

namespace Domain.Entities
{
    public class GameResult:BaseEntity
    {
        public int GameSetupId { get; set; }
        public GameSetup GameSetup { get; set; }
        public int FirstChoice { get; set; }
        public int OpenedDoorNo { get; set; }
        public int SecondChoice { get; set; }
        public bool IsWin { get; set; } 
    }
}

File: Client/src/app/shared/models/gameResult.ts
import { IGameSetup } from './gameSetup';

export interface IGameResult {
  id: number;
  gameSetupId: number;
  gameSetup: IGameSetup;
  firstChoice: number;
  openedDoorNo: number;
  secondChoice: number;
  isWin: boolean;
}
